src/index.html

```diff
@@ -6,9 +6,44 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="header-logo">
+        <img src="images/logo.png" alt="moyo-logo">
+      </a>
+      <nav class="header-nav">
+        <ul class="nav-list">
```

Comment:
Let`s apply this code style here 
![image](https://github.com/mate-academy/layout_moyo-header/assets/46566190/986a59ab-2e40-4544-b306-1eac92f048bd)